src/frontend/js/hooks/useResources/index.spec.tsx

```diff
@@ -0,0 +1,225 @@
+import fetchMock from 'fetch-mock';
```

Comment:
We should also test the behaviour when the api request fails.

Reply:
Good catch